zubhub_frontend/zubhub/src/assets/js/styles/views/create_activity/createActivityStyles.js

```diff
@@ -0,0 +1,50 @@
+export const styles = theme => ({
+  createActivityContainer: {
+    backgroundColor: '#FFCD06',
+    padding: '5vh 10%'
```

Comment:
Not a good idea to use vh for padding

Reply:
i want it to changes dynamically with screen height should i use rem as unit?